### DISCUSSION TITLE
Mmoitems Support

### DISCUSSION THREAD

**Discussion by Keopsfenks** [AlessioGr/NotQuests#228] [Feature Requests]
It would be great if Mmoitems support comes.

**Comment by AlessioGr** [comment]
> It would be great if Mmoitems support comes.

how  _exactly_ should mmoitems support look like? What function should notquests support?

**Comment by editon96** [comment]
I think he want that Iteams what are selfmade can be used in Quests as Reward or maybe needed to make that quest.

Custum Model Data Support are for that needed and maybe NBT Support I dont now maybe I write shit.

But maybe that is what he want I think that were nice when not allready there.

**Comment by PaintsplattersMC** [comment]
I agree this would be great. mmoitems/mmocore/mythiclib support. 

Ability to check if player has X mmoitem item in their inventory. If they use Y skills etc
